app/app/local.env

```diff
@@ -30,3 +30,9 @@ GITHUB_CLIENT_SECRET=
 GITHUB_APP_NAME=
 
 INFURA_USE_V3=True
+
+SENDGRID_API_KEY=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
```

Comment:
did you intend to include this?

Reply:
No, I think I missed it when doing a rebase.  I'll remove it.